src/utils.ts

```diff
@@ -53,4 +53,29 @@ function getCheckText(trivyStatus: number, dockleStatus: number): string {
     }
 
     return text;
+}
+
+export function getConfigForTable(col1_width: number, col2_width: number, col3_width: number, col4_width: number): any {
```

Comment:
low priority: Should we stick to camelCase for function parameters?

Reply:
Not using them anymore